src/main/java/de/baleipzig/iris/ui/language/MessageKeys.java

```diff
@@ -0,0 +1,9 @@
+package de.baleipzig.iris.ui.language;
+
+public class MessageKeys {
+    public static final String APPLICATION_NAME = "base.application.name";
+
+    public static final String RECOGNITION_FOO_BAR = "recognition.foo.bar";
+
+    public static final String TRAINING_NOT_FOO = "training.not.foo";
+}
```

Comment:
das kann komplett weg, da redundant. du reproduzierst hier nur die struktur im resource-file in capslock mit unterstrich, was den aufwand zum anlegen einer resource nur verdoppelt. 
ich habe kein problem damit, die resource im quellcode mit
languageHandler.getTranslation("base.application.name") 
aufzurufen. ist auch kürzer. ;)